README.md

```diff
@@ -1,6 +1,32 @@
 # AirPyllution
+A package for visualizing future, historic and current air pollution data using the [OpenWeather API](https://openweathermap.org).
 
-A package which  provides various functionalities on air pollution data.
+## Summary
+This package enables users to explore air pollution levels in locations around the world.
+Using the [Air Pollution API](https://openweathermap.org/api/air-pollution), this package provides 3 functions that help to visualise present, future and historic air pollution data.  
+
+The data returned from the API includes the polluting gases such as Carbon monoxide (CO), Nitrogen monoxide (NO), Nitrogen dioxide (NO2), Ozone (O3), Sulphur dioxide (SO2), Ammonia (NH3), and particulates (PM2.5 and PM10).
+
+Using the OpenWeatherMap API requires sign up to gain access to an API key.  
+This API key should be stored in the `config.py` file. (TO ADD)  
+For more information about API call limits and API care recommendations please visit the [OpenWeather how to start](https://openweathermap.org/appid) page.
+## Functions
+The functions are as follows:
+- `getAirPollution()`
+- `getPollutionHistory()`
+- `getPollutionForecast()`
+
+### `getAirPollution()`
+Fetches the air pollution levels based on a location (string). Based on the values of the polluting gases, this package uses the [Air Quality Index](https://en.wikipedia.org/wiki/Air_quality_index#CAQI) to determine the level of pollution for the location and produces a coloured map of the area displaying the varying regions of air quality.
+
+### `getPollutionHistory()`
+Requires a start and end date and fetches historic air pollution data for a specific location. The function returns a data frame with the values of the polluting gases over the specified date range.
+
+### `getPollutionForecast()`
```

Comment:
Does this function mean we are solving a predictive problem ? Or is this data present in the dataset we generate ?

Reply:
No, we're simply just returning the data fetched from the API as a dataframe. 
The API response for the Historical air pollution request (https://openweathermap.org/api/air-pollution) returns data in a JSON so we could create a function that turns the JSON object into a table?

Reply:
Sorry Mel , should I have mentioned the function. Im asking about `getPollutionForecast()` ?

Reply:
> @danfke @christopheralex Just an FYI - the descriptions are not set in stone - happy to change the function description/ functionality later once we're done with the POC

But as mentioned. We can go ahead and merge it as these are our preliminary assumptions/functionalities.

Reply:
Oh i see, sorry! The forecast data is also part of the API - `Forecast air pollution data` They provide us with forecasted data for the next 5 days

Reply:
Cool 👍